go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/giantswarm/chart-operator
 go 1.13
 
 require (
-	github.com/giantswarm/apiextensions v0.2.5
-	github.com/giantswarm/appcatalog v0.1.11
+	github.com/giantswarm/apiextensions v0.3.1
+	github.com/giantswarm/appcatalog v0.1.12-0.20200428072355-806bc411c715
```

Comment:
TODO: get back to real version after review.

Reply:
So you merged it without doing this now.